src/main/scala/org/hammerlab/guacamole/commands/jointcaller/kryo/Registrar.scala

```diff
@@ -0,0 +1,38 @@
+package org.hammerlab.guacamole.commands.jointcaller.kryo
+
+import com.esotericsoftware.kryo.Kryo
+import org.apache.spark.serializer.KryoRegistrator
+import org.hammerlab.guacamole.commands.jointcaller.Parameters.SomaticGenotypePolicy
+import org.hammerlab.guacamole.commands.jointcaller.annotation.{InsufficientNormal, MultiSampleAnnotations, SingleSampleAnnotations, StrandBias}
+import org.hammerlab.guacamole.commands.jointcaller.evidence.{MultiSampleMultiAlleleEvidence, MultiSampleSingleAlleleEvidence, NormalDNASingleSampleSingleAlleleEvidence, TumorDNASingleSampleSingleAlleleEvidence, TumorRNASingleSampleSingleAlleleEvidence}
+import org.hammerlab.guacamole.commands.jointcaller.{AlleleAtLocus, Input, InputCollection, Parameters}
+
+class Registrar extends KryoRegistrator {
```

Comment:
I thought about `object`ing this, but I like the consistency with other "registrators", and also having an eye towards a possible future where modules handle their own kryo registration, and can e.g. run their tests with just their registrar being instantiated, not the global guacamole one that registers everything. It's a very minor point though, either way.

any registrar that is passed to Spark must be a zero-argument class, not an object, due to [the way spark instantiates them](https://github.com/apache/spark/blob/v1.6.1/core/src/main/scala/org/apache/spark/serializer/KryoSerializer.scala#L123).